src/arrayMethodSort.js

```diff
@@ -1,11 +1,36 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
+const defaultCompare = (a, b) => {
+  if (String(a) > String(b)) {
+    return 1;
+  } else if (String(a) < String(b)) {
+    return -1;
+  }
```

Comment:
Remember about else and return.
```suggestion
  }
  
  if (String(a) < String(b)) {
    return -1;
  }
```